assets/airports/kclt.json

```diff
@@ -0,0 +1,7898 @@
+{
+    "airac": 1813,
+    "radio": {
+        "twr": "Charlotte Tower",
+        "app": "Charlotte Approach",
+        "dep": "Charlotte Departure"
+    },
+    "icao": "KCLT",
+    "iata": "CLT",
+    "magnetic_north": -7.0,
+    "ctr_radius": 70,
+    "ctr_ceiling": 16000,
+    "initial_alt": 8000,
+    "position": ["N35.2137500", "W80.9490556", "748ft"],
+    "rangeRings": {
+        "enabled": true,
+        "center": ["N35.2137500", "W80.9490556"],
+        "radius_nm": 5.0
+    },
+    "has_terrain": true,
+    "wind": {
+        "angle": 200,
+        "speed": 8
+    },
+    "arrivalRunway": "18R",
+    "departureRunway": "18C",
+    "defaultMaps": ["Base Map"],
+    "airspace": [
+        {
```

Comment:
L29 and below: missing correct indentions

Reply:
Fixed